src/renderer/components/+catalog/catalog.tsx

```diff
@@ -38,8 +38,8 @@ import { Tab, Tabs } from "../tabs";
 import { catalogCategoryRegistry } from "../../../common/catalog";
 import { CatalogAddButton } from "./catalog-add-button";
 import type { RouteComponentProps } from "react-router";
-import type { ICatalogViewRouteParam } from "./catalog.route";
 import { Notifications } from "../notifications";
+import type { ICatalogViewRouteParam } from "../../../common/routes";
```

Comment:
[nit] Forgot to remove `I` from `ICatalogViewRouteParam`? or not in the scope of this PR?